src/App.jsx

```diff
@@ -14,8 +34,23 @@ export default function App() {
           <Instructions />
         </InstructionsButton>
       </header>
+      <p>Current Player: {calculateCurrentPlayerMark(currentPlayerId)}</p>
+
       <div className="App-body">
-        <Board data={boardData} />
+        <div>
+          {isAlertVisible && (
+            <Alert
+              message="Move already made. Please choose an empty cell."
+              onClose={closeAlert}
+            />
+          )}
+        </div>
+        <Board
+          showAlert={showAlert}
+          closeAlert={closeAlert}
+          changeTurn={changeTurn}
```

Comment:
without seeing the UI in action and digging deep into the details, can I ask, why do you need to pass in the show/close alert into the updateGame function if the Alert is shown above here?